querydsl-codegen/src/main/java/com/querydsl/codegen/CodegenModule.java

```diff
@@ -56,6 +61,7 @@ protected void configure() {
         bind(EmbeddableSerializer.class);
         bind(ProjectionSerializer.class);
         bind(SupertypeSerializer.class);
+        bind(String.class);
```

Comment:
You should use named injection instead, like with the keywords, String is not a service interface